sql/pgwire/v3.go

```diff
@@ -622,8 +622,8 @@ func (c *v3Conn) sendResponse(resp sql.Response, formatCodes []formatCode, sendD
 		return c.sendCommandComplete(nil)
 	}
 	for _, result := range resp.Results {
-		if result.PErr != nil {
-			if err := c.sendError(result.PErr.String()); err != nil {
+		if result.Err != nil {
+			if err := c.sendError(result.Err.Error()); err != nil {
```

Comment:
Do we need to revert #4479 then? I gave it a LGTM last night but I wasn't aware of #2625.